lib/rucio/core/message.py

```diff
@@ -43,39 +40,45 @@ def add_message(event_type, payload, session=None):
     :param payload: The message payload. Will be persisted as JSON.
     :param session: The database session to use.
     """
-
-    services_list = REGION.get('services_list')
-    if services_list == NO_VALUE:
-        try:
-            services_list = config_get('hermes', 'services_list')
-        except (NoOptionError, NoSectionError, RuntimeError):
-            services_list = None
-        REGION.set('services_list', services_list)
-
+    try:
+        services_list = config_get('hermes', 'services_list')
+    except (NoOptionError, NoSectionError, RuntimeError):
+        services_list = 'activemq,email'
     try:
         payload = json.dumps(payload, cls=APIEncoder)
-    except TypeError as e:  # noqa: F841
-        raise InvalidObject('Invalid JSON for payload: %(e)s' % locals())
-
-    if len(payload) > 4000:
-        new_message = Message(event_type=event_type, payload='nolimit', payload_nolimit=payload, services=services_list)
-    else:
-        new_message = Message(event_type=event_type, payload=payload, services=services_list)
+    except TypeError as err:  # noqa: F841
+        raise InvalidObject('Invalid JSON for payload: %(err)s' % locals())
+
+    services = services_list.split(',')
```

Comment:
We already have a `config_get_int`, `config_get_bool`, ... function in the `common/config.py`. This here is the best opportunity to introduce a `config_get_list`.